llvm/lib/Target/DirectX/DXILOpBuilder.cpp

```diff
@@ -249,17 +309,95 @@ static FunctionType *getDXILOpFunctionType(const OpCodeProperty *Prop,
       ArgTys[0], ArrayRef<Type *>(&ArgTys[1], ArgTys.size() - 1), false);
 }
 
+/// Get index of the property from PropList valid for the most recent
+/// DXIL version not greater than DXILVer.
+/// PropList is expected to be sorted in ascending order of DXIL version.
+template <typename T>
+static int getPropIndex(const std::vector<T> PropList,
+                        const VersionTuple DXILVer) {
+  auto Size = PropList.size();
```

Comment:
`auto` doesn't help readability here (https://llvm.org/docs/CodingStandards.html#use-auto-type-deduction-to-make-code-more-readable)

```suggestion
  size_t Size = PropList.size();
```

Reply:
> `auto` doesn't help readability here (https://llvm.org/docs/CodingStandards.html#use-auto-type-deduction-to-make-code-more-readable)

Changed `auto` to `size_t`.